lib/refresh.js

```diff
@@ -12,42 +12,42 @@ var pullQueue  = new NotifyQueue();
 module.exports = {
    ///////   Issues   /////////
 
-   issue: function refreshIssue(number) {
+   issue: function refreshIssue(repo, number) {
       debug("refresh issue %s", number);
-      return gitManager.getIssue(number)
+      return gitManager.getIssue(repo, number)
       .then(pushOnQueue(issueQueue));
    },
 
    allIssues: function refreshOpenIssues() {
       debug("refresh all issues");
-      return gitManager.getAllIssues()
+      return gitManager.forEachRepo(gitManager.getAllIssues())
```

Comment:
Should this be `getAllIssues` without the `()`? Unless `getAllIssues` returns a function?